packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1067,6 +1067,35 @@ oie.registration.form.submit = Sign Up
 oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
+oie.user.profile.login = Username
```

Comment:
Same string `'Username'` is also defined L#529 and L#530:

> primaryauth.username.placeholder = Username
primaryauth.username.tooltip = Username

Created this entry here though, to be specific to this use case.